governance/README.md

```diff
@@ -47,12 +45,17 @@ Discussion happens on the [contributor strategy mailing list](https://lists.cncf
 
 Facilitators:
 
-- Josh Berkus ([@jberkus](https://github.com/jberkus)), Red Hat
-- Dawn Foster ([@geekygirldawn](https://github.com/geekygirldawn)), CHAOSS Project
+* Josh Berkus ([@jberkus](https://github.com/jberkus)), Red Hat
+* Dawn Foster ([@geekygirldawn](https://github.com/geekygirldawn)), CHAOSS Project
 
-## Directory
+## Resources
 
-Currently we have the following content in this working group:
+We provide resources to CNCF projects that are focused on implementing or enhancing governance processes. Projects can discover more about the available governance options and their impact on their project by visiting the [Governance section](https://contribute.cncf.io/maintainers/governance/) on our website.
 
-* **docs**: documents offering advice, processes, implementation details, and other information for project governance. Documents here are for the benefit of project leaders trying to improve their governance.
-* **draft-requirements**: directory of suggested changes to the CNCF requirements for project governance, with supporting materials.  Documents in this folder will be shared with the TOC when they revise the acceptance and diligence process.
+If your project requires governance templates, please explore the [CNCF project template repository](https://github.com/cncf/project-template). Projects have the option to either replicate the entire template or select specific sections to incorporate into your existing documentation.
+
+Additionally, we encourage projects to explore the ["Templates"](https://contribute.cncf.io/maintainers/templates/) section on our website. This section contains "How To" guides that explain the template's usage through examples, comprehensive instructions, and links to other projects that have inspired aspects of this initiative.
```

Comment:
```suggestion
Whether your project is creating your first governance document or looking for ways to improve on existing governance models, we encourage projects to explore the ["Templates"](https://contribute.cncf.io/maintainers/templates/) section on our website. This section contains "How To" guides that explain the governance template's usage through examples, comprehensive instructions, and links to other projects that have inspired aspects of this initiative. The templates themselves live in the [CNCF project template repository](https://github.com/cncf/project-template) repository, but we encourage projects to start with the "How To" guides on the website before diving into the templates directly.
```

I wanted to swap the order of this and combine them to talk about the how to guides first so that people don't just start with the templates.